prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1117,6 +1120,47 @@ static int rxr_ep_set_cuda_api_permitted(struct rxr_ep *ep, bool cuda_api_permit
 	return 0;
 }
 
+/**
+ * @brief set sendrecv_in_order_aligned_128_bytes flag in rxr_ep
+ * Supporting send/receive aligned 128 bytes buffer in order is more complicated than
+ * supporting RDMA write. There is no plan to support it in the near future.
+ * Therefore this function always returns -FI_EOPNOTSUPP if the user tries to enable it.
+ *
+ * @param[in,out]	ep					endpoint
+ * @param[in]		sendrecv_in_order_aligned_128_bytes	whether to enable in_order send/recv
+ *								for each 128 bytes aligned buffer
+ * @return		0 on success, -FI_EOPNOTSUPP if the option cannot be supported
+ */
```

Comment:
This is not intuitive, even with extensive explanation.

Suggest being expressive with the function name, e.g.
```
rxr_ep_validate_sendrecv_in_order_aligned_128_bytes_unset(...)
```

Reply:
The function makes sense to me, but I'm confused as to why we would want to introduce such an option with no plans for support.  I suppose because we want to distinguish it from the _RDMA variant, but RDMA variant's name seems sufficiently specific to avoid confusion.

Do we really need the sendrecv variant?

Reply:
> Do we really need the sendrecv variant?

In the future, I think we may implement this feature for sendrecv, so it is better to introduce it now.